packages/tools/devtools/devtools-example/src/test/ui-smoke-test/ChromeBrowserDevtool.test.ts

```diff
@@ -0,0 +1,56 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+/* eslint-disable spaced-comment */
+/// <reference types="jest-environment-puppeteer" />
+/// <reference types="puppeteer" />
+/// <reference types="jest" />
+
+import { retryWithEventualValue } from "@fluidframework/test-utils";
+import { globals } from "../../../jest.config";
+
+jest.setTimeout(30_000);
+describe("End to end tests", () => {
+	/**
+	 * Gets the value of the text form backed by our CollaborativeTextArea.
+	 *
+	 * @remarks Assumes there is only one `textarea` element on the page.
+	 *
+	 * @param expectedValue - The value we expect the value of the text area to be.
+	 */
+	async function getTextFormValue(expectedValue: string): Promise<string> {
+		// eslint-disable-next-line @typescript-eslint/no-unsafe-return
+		return retryWithEventualValue(
+			/* callback: */ async () =>
+				page.evaluate(() => {
+					const divs = document.querySelectorAll(".textarea");
+					const textAreaElements = divs[0].querySelectorAll("textarea");
+					const textarea = textAreaElements[0];
+					return textarea?.value;
+				}),
+			/* check: */ (actualValue) => actualValue === expectedValue,
+			/* defaultValue: */ "",
+		);
+	}
+
+	beforeAll(async () => {
+		// Wait for the page to load first before running any tests
+		// so this time isn't attributed to the first test
+		await page.goto(globals.PATH, { waitUntil: "load", timeout: 0 });
```

Comment:
We have a `page.goto()` in `beforeEach()` as well, so _some_ navigation time still gets attributed to each test. Is this to handle the first load, which might be longer because of webpack dev server / similar?

Reply:
This is for open a new page before each test